src/session.test.js

```diff
@@ -75,4 +76,8 @@ describe("session cases", () => {
       stickySessionId: expect.any(String),
     });
   });
+
+  it("wasShopperInsightsUsed should return false if shopperInsights doesn't exist in the state", () => {
```

Comment:
we should a true case as well

Reply:
Definitely. 👍   I was having some issues with mocking some of the storage functions before my vacation & realized all the other functions only really have negative testing.  But let me see if a fresh brain can come up with something better now 😄

Reply:
Haha actually, it was pretty easy. See.. fresh brain 🧠

Reply:
fresh brains solve most things